android/src/com/frostwire/android/gui/SearchEngine.java

```diff
@@ -247,6 +248,14 @@ public SearchPerformer getPerformer(long token, String keywords) {
         }
     };
 
+    public static final SearchEngine ONE337X = new SearchEngine("1337x", Constants.PREF_KEY_SEARCH_USE_ONE337X) {
+        @Override
+        public SearchPerformer getPerformer(long token, String keywords) {
+            return new One337xSearchPerformer("www.1377x.to", token, keywords, DEFAULT_TIMEOUT);
+            // change www.1377x.to to other like www.1337x.gd, www.1337x.to or other proxy site etc
```

Comment:
take a look at how it's done for TPB, and the `isReady()` method.

There's a class that given a bunch of domains it picks the best mirror depending on the speed of the responses.